### PR TITLE
Add WowUp v2.0.1

### PR DESCRIPTION
**Pull request by ghost**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/wowup.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  appcast "https://github.com/wowup/wowup/releases.atom"
+  name "WowUp"
+  desc "World of Warcraft Addon manager"
+  homepage "https://wowup.io/"
+
+  auto_updates true
+
+  app "WowUp.app"
+
+  uninstall quit: [
```
Comment (vitorgalvao) [inline Casks/wowup.rb:16]: Doesn’t need to be an array.

File: Casks/wowup.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "WowUp",
```
Comment (vitorgalvao) [inline Casks/wowup.rb:17]: This takes a bundle ID, not an app name. Run `mdls -raw -name kMDItemCFBundleIdentifier APP_PATH` to get it.

File: Casks/wowup.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  ]
+
+  zap trash: [
+    "~/Library/Application Support/WowUp",
+    "~/Library/Logs/WowUp",
+    "~/Library/Preferences/io.wowup.jliddev.plist",
+    "~/Library/Saved\ Application\ State/io.wowup.jliddev.savedState/",
```
Comment (vitorgalvao) [inline Casks/wowup.rb:24]: Remove the `\`. The string is quoted.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
Welcome! Looks good, only requiring minor fixes.

File: Casks/wowup.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+cask "wowup" do
+  version "2.0.0"
```
Comment (vitorgalvao) [inline Casks/wowup.rb:2]: It has since been updated to `2.0.1`.

**Review by vitorgalvao — DISMISSED** [review]

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @MKaysen.

**Review by miccal — APPROVED** [review]

**Pull request merged**